src/main/java/frc/robot/util/ShuffleBoard/Tabs/AutoTab.java

```diff
@@ -171,10 +171,10 @@ public void createEntries(){
         
 
 //         Repleacement Auto, don't delete
-//         autoCommand.addOption("lol", new FollowPathCommand("lol",true, drive));
+        autoCommand.addOption("lol", new FollowPathCommand("lol",true, drive));
```

Comment:
What was the point of not commenting this out? I thought we didn't want to add any auto options on Shuffleboard. Why not just keep in the DoNothing, if we just want something to show up? Not sure if I'm understanding correctly.

Reply:
It says not to delete it. What was the point of this path, and can we delete it now?